Extensions/TileMap/JsExtension.js

```diff
@@ -951,10 +952,14 @@ module.exports = {
         instance,
         associatedObject,
         pixiContainer,
-        pixiResourcesLoader
+        pixiResourcesLoader,
+        pixiRenderer
       );
 
-      this.tileMapPixiObject = new Tilemap.CompositeRectTileLayer(0);
+      Tilemap.settings.use32bitIndex = true;
```

Comment:
I'm probably missing something, but why is this line present twice? in the extension and in the pixi-object-renderer?

Reply:
This is because the renderer is used in the IDE and at runtime.

Reply:
Oh yeah, there're different instances. 👍 thanks